### PR TITLE
DXISSUE-47: Update readme and copyright information, and make formatt…

### PR DESCRIPTION
**Pull request by niemczyk-m**
This is to update readme information and copyright information in that readme. I also made formatting adjustments to use markdown, so the readme and changelog are more clear. Some of the links on readme didn't work, so I removed them.

### CODE REVIEW

**Review by jjjhhhlll — APPROVED** [review]

**Comment by niemczyk-m** [comment]
@jjjhhhlll Do you think you can merge it for me after approval? As I don't have write access to the repo to merge the PR to the main branch.

**Pull request merged**
